frontend/pages/SoftwarePage/SoftwarePage.tsx

```diff
@@ -147,6 +149,8 @@ const SoftwarePage = ({ children, router, location }: ISoftwarePageProps) => {
     queryParams && queryParams.page
       ? parseInt(queryParams.page, 10)
       : DEFAULT_PAGE;
+  const platform =
```

Comment:
I think we can re-write this as `const platform = queryParams?.platform || "all";`

Reply:
Revised for the platform selection. Not touching the others because e.g. page needs special logic anyway that we can't drop to just nullsafe.